src/config.js

```diff
@@ -19,6 +19,12 @@ const config = convict({
             default: null,
             format:  String,
             env:     'JENKINS_API_BASIC_AUTH_PASSWORD'
+        },
+        customCa: {
+            doc:     '(Optional) CA file to use for tls authentication',
+            default: '',
```

Comment:
@plouc Because I want to make sure it is optional. If I have it as null, and do not configure the `JENKINS_API_CUSTOM_CA` or comment it out, then I cannot start the server as convict will try to validate the jenkins config:

`error: jenkins.customCa: must be of type String`